src/types.tsx

```diff
@@ -217,7 +206,7 @@ export interface FormikConfig<Values> extends FormikSharedConfig {
  * of <Formik/>.
  */
 export type FormikProps<Values> = FormikSharedConfig &
-  FormikState<Values> &
+  Formik.State<Values> &
   FormikActions<Values> &
```

Comment:
We could probably move several of these types under a `Formik` namespace for consistency.